docs/TUTORIAL.md

```diff
@@ -92,60 +94,84 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     As we said before fogg works by generating code (terraform, make and bash) and the general workflow is–
 
-    1. edit `fogg.json`
+    1. edit `fogg.yml` 
+        - Add an `account_id` to the aws provider
```

Comment:
is this just an example of an edit (and if so is it useful)? or does a user always have to do this?

Reply:
They always have to add an account id